vividus-plugin-kafka/src/main/java/org/vividus/steps/kafka/KafkaSteps.java

```diff
@@ -135,25 +135,25 @@ public void sendData(String data, String producerKey, String topic) throws Inter
      * @param consumerKey The key of the producer configuration
      * @param topics      The comma-separated set of topics to listen
      */
-    @When("I start consuming messages from `$consumerKey` Kafka topics `$topics`")
+    @When("I start consuming events from `$consumerKey` Kafka topics `$topics`")
     public void startKafkaListener(String consumerKey, Set<String> topics)
     {
         stopListener(getListeners().remove(consumerKey), false);
-        BlockingQueue<String> messageQueue = new LinkedBlockingDeque<>();
-        testContext.get(MESSAGES_KEY, HashMap::new).put(consumerKey, messageQueue);
+        BlockingQueue<String> eventValueQueue = new LinkedBlockingDeque<>();
+        testContext.get(EVENTS_KEY, HashMap::new).put(consumerKey, eventValueQueue);
         ContainerProperties containerProperties = new ContainerProperties(topics.toArray(new String[0]));
         containerProperties.setMessageListener(
-                (MessageListener<String, String>) data -> messageQueue.add(data.value()));
+                (MessageListener<String, String>) data -> eventValueQueue.add(data.value()));
         GenericMessageListenerContainer<String, String> container = new KafkaMessageListenerContainer<>(
                 consumerFactories.get(consumerKey), containerProperties);
         container.start();
         getListeners().put(consumerKey, container);
 
-        LOGGER.info("Kafka message listener is started");
+        LOGGER.info("Kafka event listener is started");
     }
 
     /**
-     * Waits until the count of the consumed messaged (from the consumer start or after the last draining operation)
+     * Waits until the count of the consumed messages (from the consumer start or after the last draining operation)
```

Comment:
```suggestion
     * Waits until the count of the consumed events (from the consumer start or after the last draining operation)
```

Reply:
I keep the docs aligned with the step, the step is about messages so the docs are about messages as well